tips/TIP-0020/tip-0020.md

```diff
@@ -0,0 +1,501 @@
+---
+tip: 20
+title: Transaction Payload with TIP-18 Output Types
+description: Add output types, unlocks, and output features from TIP-18 into Transaction Payload
+author: Levente Pap (@lzpap) <levente.pap@iota.org>
+discussions-to: https://github.com/iotaledger/tips/pull/40
+status: Draft
+type: Standards
+layer: Core
+created: 2021-11-18
+requires: TIP-7 and TIP-18
+---
+
+# Summary
+
+This TIP proposes a UTXO-based transaction structure consisting of all the inputs and outputs of a transfer. Specifically, this TIP defines a transaction payload for _blocks_ described in [TIP-24](../TIP-0024/tip-0024.md).
+
+# Motivation
+
+There are several options on how transfers of IOTA coins can be embedded into the actual vertices of the Tangle. In the legacy IOTA protocol, each vertex corresponds to a single transaction, i.e. an input or an output. In order to support atomic transfers, input/output transactions have to be grouped into a so-called bundle and then either the entire bundle is applied to the ledger or none of its transactions.
```

Comment:
Do we still have the same motivation like in TIP-7?
Not sure if we need to talk about the old legacy IOTA here, if this was already done in the TIP that got replaced by this one.
Might be a bit confusing to the reader that doesn't know the old tech IMO.